tests/foreman/api/test_contentview.py

```diff
@@ -754,7 +754,7 @@ def test_positive_reboot_recover_cv_publish(self, target_sat):
 
         :CaseAutomation: Automated
         """
-        org = entities.Organization().create()
```

Comment:
Any reason why you're only updating some test functions in this module? I'd say go for all of them so that you can remove the import.

Reply:
currently planned to update only destructive tests.